pyeudiw/trust/__init__.py

```diff
@@ -13,6 +13,9 @@
     MissingTrustType
 )
 
+import pyeudiw.metadata.policy as pcl
+
+#from pyeudiw.metadata.policy import TrustChainPolicy, combine
```

Comment:
remove this comment if not of interest anymore

Reply:
Resolved in [c062b35](https://github.com/italia/eudi-wallet-it-python/pull/188/commits/c062b354e94337a19c652e5fdc50240b3a8422f1)